parcel_test.go

```diff
@@ -31,57 +32,120 @@ func getTestParcel() Parcel {
 // TestAddGetDelete проверяет добавление, получение и удаление посылки
 func TestAddGetDelete(t *testing.T) {
 	// prepare
-	db, err := // настройте подключение к БД
+	// настройте подключение к БД
+	db, err := sql.Open("sqlite", "tracker.db")
+	if err != nil {
+		fmt.Println(err)
+		return
+	}
+	defer db.Close()
 	store := NewParcelStore(db)
 	parcel := getTestParcel()
 
 	// add
 	// добавьте новую посылку в БД, убедитесь в отсутствии ошибки и наличии идентификатора
+	id, err := store.Add(parcel)
 
+	require.NoError(t, err)
+	require.NotEmpty(t, id)
 	// get
 	// получите только что добавленную посылку, убедитесь в отсутствии ошибки
+	got, err := store.Get(id)
+
+	require.NoError(t, err)
+
 	// проверьте, что значения всех полей в полученном объекте совпадают со значениями полей в переменной parcel
+	require.Equal(t, parcel.Address, got.Address)
+	require.Equal(t, parcel.Client, got.Client)
+	require.Equal(t, parcel.CreatedAt, got.CreatedAt)
+	require.Equal(t, parcel.Status, got.Status)
 
 	// delete
 	// удалите добавленную посылку, убедитесь в отсутствии ошибки
+	err = store.Delete(got.Number)
+	require.NoError(t, err)
+
 	// проверьте, что посылку больше нельзя получить из БД
+	got1, err := store.Get(got.Number)
+
+	require.Equal(t, sql.ErrNoRows, err)
```

Comment:
Для сравнивания ошибок нужно использовать специализированную функцию
```suggestion
	require.ErrorIs(t, err, sql.ErrNoRows)
```